src/apis/Chocolatey.js

```diff
@@ -0,0 +1,14 @@
+const { APIWrapper } = require('../')
+const axios = require('axios')
+
+module.exports = class Chocolatey extends APIWrapper {
+  constructor () {
+    super({
+      name: 'chocolatey'
+    })
+  }
+
+  async search (query) {
+    return axios.get(`https://chocolatey.org/api/v2/Search()?$filter=IsLatestVersion&$skip=0&$top=10&searchTerm=${encodeURIComponent(`'${query}'`)}&targetFramework=%27%27&includePrerelease=false`)
```

Comment:
Use the `params` options to make the request more readable.

Reply:
https://github.com/SwitchbladeBot/switchblade/pull/1166/commits/bce310460b66e08b115588316de7fab69bd7b1e7